resources/css/main.css

```diff
@@ -1,37 +1,499 @@
 *,
 *::before,
 *::after {
-  font-family: georgia, "times new roman", serif;
   box-sizing: border-box;
   margin: 0;
   padding: 0;
 }
 
 /** Typography **/
-h1 {
-  font-size: 40px;
-  font-style: italic;
-  font-weight: 700;
-  color: #121212;
+main h2 {
+  font-family: georgia, serif;
+  font-size: 20px;
+  display: inline-block;
+  margin-bottom: 20px;
+  margin-top: 40px;
 }
 
 p {
   font-size: 20px;
   font-weight: normal;
+  font-family: georgia, "times new roman", serif;
+  color: rgb(51, 51, 51);
   margin-bottom: 20px;
   line-height: 1.5em;
 }
 
+a {
+  font-family: inherit;
+}
+
 /** Multimedia **/
 img {
   max-width: 100%;
 }
 
-/** Block elements **/
-main {
-  padding: 0 1.5em;
+/** Components **/
+.image {
+  width: 720px;
+  margin: 40px auto;
 }
 
-figure {
+.image-caption {
+  color: rgb(102, 102, 102);
+  font-family: georgia, "times new roman", serif;
+  font-size: 15px;
+  font-weight: 400;
   width: 70%;
+  margin-top: 8px;
+  line-height: 20px;
+}
+
+.image .image-caption {
+  width: 100%;
+}
+
+.image-author {
+  color: rgb(136, 136, 136);
+  font-family: georgia, "times new roman", serif;
+  font-size: 13px;
+  display: inline-block;
+  padding-left: 2px;
+  letter-spacing: 0.13px;
+  font-weight: 400;
+  vertical-align: baseline;
+}
+
+.wrap {
+  width: 600px;
+  margin: 0 auto;
+}
+
+/** Button **/
+.topbar-button {
+  background: #567b94;
+  color: white;
+  text-transform: uppercase;
+  font-weight: 600;
+  font-family: helvetica, arial, sans-serif;
+  font-size: 10px;
+  letter-spacing: 0.5px;
+  outline: 0;
+  border: 0;
+  padding: 11px 12px 10px;
+  border-radius: 3px;
+  margin-right: 8px;
+}
+
+.topbar-button:hover,
+.topbar-button:focus {
+  background: rgb(50, 104, 145);
+}
+
+.topbar-settings {
+  color: black;
+  font-size: 12px;
+  text-decoration: none;
+}
+
+.topbar-caret {
+  width: 8px;
+}
+
+/** Topbar **/
+.topbar {
+  background: white;
+  border-bottom: 1px solid #e2e2e2;
+  padding: 6px 1.5%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  z-index: 10;
+}
+
+.topbar-icon {
+  width: 15px;
+  border: none;
+  background: none;
+  padding: 8px;
+  margin-right: 8px;
+  box-sizing: content-box;
+  border-radius: 3px;
+}
+
+.topbar-icon:hover,
+.topbar-icon:focus {
+  background: #f7f7f7;
+}
+
+.topbar-item {
+  font-family: helvetica, arial, sans-serif;
+  display: inline-block;
+}
+
+.topbar-topic {
+  color: black;
+  font-weight: 600;
+  font-size: 14px;
+  text-transform: uppercase;
+  text-decoration: none;
+}
+
+.topbar-topic:hover,
+.topbar-topic:focus {
+  text-decoration: underline;
+}
+
+.topbar-logo {
+  width: 195px;
+  position: relative;
+  top: 5px;
+}
+
+/** Main section **/
+.section-flex {
+  display: flex;
+}
+
+.section-aside {
+  padding-top: 600px;
+  padding-left: 90px;
+}
+
+.section-content {
+  width: 90%;
+  padding-left: 300px;
+}
+
+/* Main content */
+#article-content {
```

Comment:
- [ ] Don't use IDs in selectors.

Reply:
We fixed this by using a class instead. Thank you!